WooCommerce/Classes/ViewRelated/Orders/Cells/OrderTableViewCell.swift

```diff
@@ -91,6 +91,18 @@ final class OrderTableViewCell: UITableViewCell & SearchResultCell {
         super.prepareForReuse()
         paymentStatusLabel.layer.borderColor = UIColor.clear.cgColor
     }
+
+    override func updateConfiguration(using state: UICellConfigurationState) {
+        guard ServiceLocator.featureFlagService.isFeatureFlagEnabled(.splitViewInOrdersTab) else {
+            return
+        }
+
+        var backgroundConfiguration = UIBackgroundConfiguration.listPlainCell().updated(for: state)
```

Comment:
```suggestion
        var backgroundConfiguration: UIBackgroundConfiguration
        if #available(iOS 16.0, *) {
            backgroundConfiguration = defaultBackgroundConfiguration().updated(for: state)
        } else {
            backgroundConfiguration = UIBackgroundConfiguration.listPlainCell().updated(for: state)
        }
```

[Apple suggest](https://developer.apple.com/documentation/uikit/uitableviewcell/3601057-contentconfiguration#discussion) using `defaultContentConfiguration`, it seems sensible to also use the `defaultBackgroundConfiguration` in this case. But it's only available from iOS 16 onwards.

Reply:
Nice! Updated: https://github.com/woocommerce/woocommerce-ios/pull/11621/commits/8e70059b1b6e0db84b713ec0f493e9e7ddecf351

Reply:
Probably worth doing the same in `configureBackground` then, if we're keeping that function 😊

Reply:
Yeah! since I merged this already I've opened a new PR here: https://github.com/woocommerce/woocommerce-ios/pull/11691